packages/govuk-frontend/src/govuk/init.mjs

```diff
@@ -19,14 +19,23 @@ import { SupportError } from './errors/index.mjs'
  * Use the `data-module` attributes to find, instantiate and init all of the
  * components provided as part of GOV.UK Frontend.
  *
- * @param {Config & { scope?: Element }} [config] - Config for all components (with optional scope)
+ * @param {Config & { scope?: Element } & { onError?: OnErrorCallback<CompatibleClass> }} [config] - Config for all components (with optional scope)
```

Comment:
**suggestion** Could this be grouped in a single object with all the non-Config options?

```suggestion
 * @param {Config & { scope?: Element, onError?: OnErrorCallback<CompatibleClass> }} [config] - Config for all components (with optional scope)
```